### PR TITLE
Designs: add empty proposal state and new activity icons for delegate voting

### PR DESCRIPTION
**Pull request by 0xGabi**
Screen Shot 2021-11-29 at 6 52 35 PM

### CODE REVIEW

File: src/assets/activity/light/delegateVoting.svg [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (rperez89) [inline src/assets/activity/light/delegateVoting.svg:1]: let's optimize this

File: src/assets/activity/light/voteOnBehalfOf.svg [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (rperez89) [inline src/assets/activity/light/voteOnBehalfOf.svg:1]: also this one

File: src/components/Stepper/Step/Illustration.js [hunk 3]
```diff
@@ -35,7 +36,7 @@ function Illustration({ status, index }) {
             color: ${theme.positiveContent};
           `}
         >
-          <IconBlock size="large" />
+          <img src={blockIcon} height={48} width={48} />
```
Comment (rperez89) [inline src/components/Stepper/Step/Illustration.js:39]: just to double check, did we check that this change is not making the other assets display in a weird way?

File: src/components/Garden/Feed/ProposalsList.js [hunk 4]
```diff
@@ -7,6 +7,8 @@ import {
   useTheme,
   useViewport,
 } from '@1hive/1hive-ui'
+
+import noProposalsYetLogo from '@assets/noProposalsYet.svg'
```
Comment (fabriziovigevani) [inline src/components/Garden/Feed/ProposalsList.js:11]: Let´s move it alogn side the svg imports albhabetized

File: src/components/Stepper/Step/Illustration.js [hunk 5]
```diff
@@ -11,6 +11,7 @@ import {
 import signRequestSuccessIllustration from '@assets/signRequestSuccess.svg'
 import signRequestFailIllustration from '@assets/signRequestFail.svg'
 import trxBeingMinedIllustration from '@assets/trxBeingMined.svg'
+import blockIcon from '@assets/blockIcon.svg'
```
Comment (fabriziovigevani) [inline src/components/Stepper/Step/Illustration.js:14]: Let´s move it to be the first svg import

File: src/utils/asset-utils.js [hunk 6]
```diff
@@ -17,6 +18,7 @@ import StakeToProposalLight from '@assets/activity/light/stakeToProposal.svg'
 import UnwrapTokenLigh from '@assets/activity/light/unwrapToken.svg'
 import UpdateOracleLight from '@assets/activity/light/updateOracle.svg'
 import VoteOnDecisionLight from '@assets/activity/light/voteOnDecision.svg'
+import VoteOnBehalfOfLight from '@assets/activity/light/voteOnBehalfOf.svg'
```
Comment (fabriziovigevani) [inline src/utils/asset-utils.js:21]: Let´s swap the imports of position

**Review by fabriziovigevani — COMMENTED** [review]
Nice, some nitpicks

**Pull request merged**